### PR TITLE
add terms and privacy language

### LINKED ISSUE #39: Add list of links in page footer to privacy doco
From [acceptance criteria](https://github.com/mozilla/ThemesRFun/blob/master/docs/acceptance.md):

> - [ ] It should have a list of mandatory links at the bottom of the page including any required privacy documentation.

### PR DESCRIPTION
**Pull request by johngruen**
Fixes #39

### CODE REVIEW

File: src/web/lib/components/SavedThemeSelector/index.scss [hunk 1]
```diff
@@ -1 +1,20 @@
 @import '../../common.scss';
+
+.saved-theme-selector {
```
Comment (johngruen) [inline src/web/lib/components/SavedThemeSelector/index.scss]: oh, actually i want to back this out...

File: src/web/lib/components/Modal/index.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    toggleModal({display: false});
+  };
+  return (
+    <div
+      className="modal"
+      onClick={handleToggle}
+    >
+      <div className="modal__content">
+        <button className="modal__toggle" onClick={handleToggle}>
+          <img src={iconClose} />
```
Comment (pdehaan) [inline src/web/lib/components/Modal/index.js]: Does this need an `alt=""` or `aria-label` or anything?

File: src/web/lib/components/Modal/index.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
+import React from "react";
+
+import iconClose from "../../../../images/close-16.svg";
+import "./index.scss";
+
+export const SavedThemeSelector = ({ toggleModal, children }) => {
```
Comment (pdehaan) [inline src/web/lib/components/Modal/index.js]: Forgive my endless ignorance, but is "SavedThemeSelector" legit here (and default export below on L26)? Or should this be something like "Modal" and this was just copy-pasta?

Comment (johngruen) [reply src/web/lib/components/Modal/index.js]: Oh yeah, that should be Modal...total copypasta

**Comment by johngruen** [comment]
Okay i reworked this again to refactor and tidy up the app component

File: src/web/lib/components/App/index.js [hunk 4]
```diff
@@ -84,41 +83,30 @@ export const AppComponent = ({
...
-                savedThemes,
-                generateThemeKey: storage.generateThemeKey,
-                putTheme: storage.putTheme,
-                userHasEdited,
-                modifiedSinceSave
-              }}
-            />
-          </div>
-        </BrowserPreview>
+        <ThemeBuilder
```
Comment (johngruen) [inline src/web/lib/components/App/index.js]: Pulled this out to make the app less spaghetti

File: src/web/lib/components/AppFooter/index.js [hunk 5]
```diff
@@ -9,74 +9,78 @@ import iconTwitter from "./twitter-logo.svg";
...
-        aria-label="Firefox Color GitHub"
-        title="GitHub"
-      >
-        <ReactSVG path={iconGH} className="app-footer__social-logo" />
-      </a>
-    </nav>
-  </footer>
-);
+export const AppFooter = ({ setDisplayLegalModal }) => {
+  const toggleModal = name => {
```
Comment (johngruen) [inline src/web/lib/components/AppFooter/index.js]: Added modal controls to the the footer so we can trigger appropriate actions on terms/privacy clicks

File: src/web/lib/components/TermsPrivacyModal/index.js [hunk 6]
```diff
@@ -0,0 +1,32 @@
+import React from "react";
```
Comment (johngruen) [inline src/web/lib/components/TermsPrivacyModal/index.js]: Made this into it's own component to keep the App component uncluttered with strings

**Comment by johngruen** [comment]
@meandavejustice could you give me a review on this one?

**Pull request merged**